streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -226,8 +226,8 @@ private StreamThread createStreamThread(@SuppressWarnings("SameParameterValue")
             0,
             stateDirectory,
             new MockStateRestoreListener(),
-            threadIdx
-        );
+            threadIdx,
+            new AtomicInteger());
```

Comment:
The closing parenthesis should go to new line:
```suggestion
            new AtomicInteger()
        );
```
Please check all your changes in this file.

Reply:
turns out the auto refactoring does not work with new params at the end of methods. fixed.